CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Leaflet Changelog
 * Fix `Class.extend` to not modify source props object ([#6766](https://github.com/Leaflet/Leaflet/pull/6766) by [@johnd0e](https://github.com/johnd0e))
 * Improve `Browser.touch` touch devices detection ([#7029](https://github.com/Leaflet/Leaflet/pull/7029) by [@johnd0e](https://github.com/johnd0e))
 * Get rid of legacy Android hacks ([#7022](https://github.com/Leaflet/Leaflet/pull/7022) by [@johnd0e](https://github.com/johnd0e))
+* Allow fonts to respect users' browser settings by making the `font-size` _relative_ to the map container. (You can change the font size on `leaflet-container` to adjust it if needed.) ([#7800](https://github.com/Leaflet/Leaflet/pull/7800) by [@Chandu-4444](https://github.com/Chandu-4444))
```

Comment:
There already exists an 1.8.0-beta.0 entry for this change, under "Accessibility". I think it's fine to keep it, but let me know if I should delete it:

https://github.com/Leaflet/Leaflet/blob/7847cc6913c296fb7e409e086a57366237c42728/CHANGELOG.md?plain=1#L101

Reply:
> You can change the font size on `leaflet-container` to adjust it if needed

This was really enabled in https://github.com/Leaflet/Leaflet/pull/8057. But I don't think we need even more entries on this topic...